lizmap/app/system/mainconfig.ini.php

```diff
@@ -26,6 +26,10 @@
 ; the locale to fallback when the asked string doesn't exist in the current locale
 fallbackLocale=en_US
 
+[minimumRequiredVersion]
```

Comment:
Please add a concise comment above with a `;` to explain the purpose of these new variables